### ISSUE TITLE
No Transfer Crew option on HAB parts

### ISSUE THREAD

**Issue by taterkerman** [shadowmage45/SSTULabs#389]
HAB-A1 issue. I transferred 1 crew to the inflatable, and had 2 issues. One, the contract did not satisfy (5 kerbals, where the rest of the station was a Soyuz). Two, there is no transfer crew option on the right click (Bob is in there, marked as crew in the save). 

Log threw this:

LOG 22:11:18.217] Setting crew capacity to: 4 current: 0
[WRN 22:11:30.608] InternalModel error: Part Crew capacity is 2, but 3 seats are defined in internal model
[WRN 22:11:30.613] InternalModel error: Part Crew capacity is 2, but 3 seats are defined in internal model
[WRN 22:11:30.613] [Part]: SSTU-ST-HAB-A1 holds crew but has no interior model defined!

That log entry happened when I tried a crew report (since there was no transfer option).

This is basically what @mechanicH had reported earlier---but there is no "Transfer Crew" option available in the right click, but Bob was marked as crew for the A1 (I checked the persistent file).

I tested all the HAB inflatables, they all do this. I realized that when this was reported before, I had no issues, but I was on 1.2, not 1.2.1... I tried restarting KSP, as well, no dice. I have a few other mods in there right now, near future construction, KER, EVE, Chatterer, and TRAILS (just the Gemini pod stuff, no tanks).

**Comment by shadowmage45** [comment]
Hmm...

Will have to do some investigate.  Looks like I'm still using 1.2.0 in my dev setup as well; so perhaps the stock code broke something in that update.

**Comment by taterkerman** [comment]
Yeah, that was my concern, that it might be 1.2.1 related. I failed to notice the .1 update, then just installed it a couple days ago. That said, at the forum, mechanicH somehow resolved the problem, so that discussion faded out.

I've been trying to push through a career play, because it gets me doing stuff I might not ordinarily do in sandbox testing (plus the whole upgrade mechanic). Slightly off topic, but for the part upgrades (like the MFT diameter changes), they incur a COST, and you have to explicitly go into the R&D and pay for them, because unlike other parts, they already exist in the VAB, so they are not grayed out---and they stick at the last paid for upgrade, with no option to upgrade within the VAB. I bet this is an oversight in the stock code (sort of like the @@#!$ apply upgrades in Sandbox) since Squad has not actually played with "upgrades."

**Comment by taterkerman** [comment]
(the thing about upgrade cost means that we might see more posts wondering why they unlocked a tech tree node, and their diameter increase is not available).

**Comment by taterkerman** [comment]
The plot thickens. I loaded up the save from last night... and the contract satisfied as soon as I was in the scene, and the Transfer Crew button appeared.

**Comment by shadowmage45** [comment]
Hehe, at least its a proper mystery :)

I'll be doing some investigation into this as soon as I have the wheels stuff on to the next stage.

**Comment by taterkerman** [comment]
I tried again, this time with verbose logging, and got exactly the same log stuff upon transferring crew as I posted above. Easy to reproduce. Spawn craft with a rocket part tank to inflate, and a crew capsule to transfer from to pad. Inflate hab, transfer crew.

Map mode shows transferred crew belonging to the inflated hab in question, but no transfer option from the hab is available,
